var/spack/repos/builtin/packages/ultraliser/package.py

```diff
@@ -17,6 +17,6 @@ class Ultraliser(CMakePackage):
     depends_on('libtiff')
     depends_on('ilmbase')
     depends_on('vtk~mpi')
-    depends_on('hdf5+hl+cxx~mpi')
+    depends_on('hdf5+hl+cxx')
```

Comment:
This solves a conflict with h5py (+mpi) which is a requirement of py-voxcell.

Reply:
this is lgtm